### DISCUSSION TITLE
WG Core 2024-09-10

### DISCUSSION THREAD

**Discussion by emiltin** [rsmp-nordic/community#87] [Working Groups]
## Participants
Marc, Toni, Emi. Henri, Matti,

## Testhub
La Semaforica TECSEN now has their Cartesio TLC and their TMacs system on the TestHub.
Henri has been in touch with Cross, and after completing their RSMP implementation they might be interested in connecting to the test hub.
We will work on expanding the way test results are shown on the RSMP Nordic website.

## 3.3 changes
Alarms to multiple supervisor systems https://github.com/rsmp-nordic/rsmp_core/issues/128
In Finland and France RSMP is now used to acknowledge alarms., which would be interesting to explore. Why is this the case?
https://rsmp-nordic.github.io/rsmp_core/3.2.2/applicability/basic_structure.html#alarm-status
If alarms suspension/blocking is not used, perhaps it's something we should consider for RSMP 4.

**Comment by marcgarba** [comment]
It seems it was said "Protocols in Finland (RSMP) and in France (actual historic protocols used), are noT used  to ack alarms". 
(acknowledgment alarms management only done on supervisions)
(for the case of France, the equipments does not have this notion of ack)
